modules/account-lib/test/unit/coin/dot/transaction.ts

```diff
@@ -1,8 +1,17 @@
-import { coins } from '@bitgo/statics';
+import { BaseCoin as CoinConfig, coins } from '@bitgo/statics';
 import should from 'should';
-import { KeyPair, Transaction, TransferBuilder } from '../../../../src/coin/dot';
+import { register } from '../../../../src';
+import { TransactionType } from '../../../../src/coin/baseCoin';
+import { KeyPair, Transaction, TransactionBuilderFactory, TransferBuilder } from '../../../../src/coin/dot';
 import * as DotResources from '../../../resources/dot';
-import { buildTestConfig } from './transactionBuilder/base';
+import { buildTestConfig, TestDotNetwork, TEST_NETWORK_DATA } from './transactionBuilder/base';
+
+class StubTransactionBuilderFactory extends TransactionBuilderFactory {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    const testNetwork = { ..._coinConfig.network, ...TEST_NETWORK_DATA } as TestDotNetwork;
+    super({ ..._coinConfig, network: testNetwork });
+  }
+}
 
 describe('Dot Transaction', () => {
```

Comment:
If these tests are commented, do the unit tests pass? Are we able to single point if one of these tests is the culprit?
There are only 7 tests initializing a tx here.

Reply:
Yes, the test suite passes if we comment out the explain transaction tests.